app/src/services/compound_abi.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint no-use-before-define: 0 */  // --> OFF
```

Comment:
Moved the file to abi directory, I think prettifying makes the file larger and since we are not planning to edit the ABIs anyways, maybe just leave it as is and replace it with newer ABIs as Compound release new upgrades.